src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/package.json

```diff
@@ -3,40 +3,55 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap": "^4.1.3",
+    "bootstrap": "4.2.1",
+    "connected-react-router": "6.2.1",
```

Comment:
This is a new package @Eilon

Reply:
Go ahead. We'll use Component Governance to register it.